erica/application/JobService/job_service_factory.py

```diff
@@ -0,0 +1,41 @@
+
+from typing import Callable, Type
+
+from opyoid import Injector
+from erica.api.ApiModule import ApiModule
+from erica.application.EricRequestProcessing.requests_controller import EricaRequestController, UnlockCodeActivationRequestController
+
+from erica.application.FreischaltCode.FreischaltCode import BaseDto, FreischaltCodeActivateDto
+from erica.application.FreischaltCode.Jobs.jobs import activate_freischalt_code
+from erica.application.JobService.job_service import JobService, JobServiceInterface
+
+injectors = []
+
+
+def _freischalt_code_activation_injector(dto: FreischaltCodeActivateDto):
+    _freischalt_code_activation_injector.handle_type = FreischaltCodeActivateDto
+    
+    module = ApiModule()
```

Comment:
Layered architecture shouldn't be affected, what means, the application layer shouldn't know anything about the api layer.

Reply:
You mean because of the ApiModule? We could also use the ApplicationModule here instead or should we move the service factory to the api layer?

Reply:
fixed